CHANGELOG.md

```diff
@@ -19,6 +19,9 @@
 - `[jest-leak-detector]` Make leak-detector more aggressive when running GC ([#14526](https://github.com/jestjs/jest/pull/14526))
 - `[jest-util]` Make sure `isInteractive` works in a browser ([#14552](https://github.com/jestjs/jest/pull/14552))
 - `[pretty-format]` [**BREAKING**] Print `ArrayBuffer` and `DataView` correctly ([#14290](https://github.com/facebook/jest/pull/14290))
+- `[jest-cli]` When specifying paths on the command line, only match against the relative paths of the test files ([#12519](https://github.com/facebook/jest/pull/12519))
+  - [**BREAKING**] Changes `testPathPattern` configuration option to `testPathPatterns`, which now takes a list of patterns instead of the regex.
+  - [**BREAKING**] `--testPathPattern` is now `--testPathPatterns`
```

Comment:
I assume most people just do `jest some/path` and not actually specify `testPathPattern`, but for people that _do_, can we print a deprecation warning for them?

Reply:
I made it show the following message and abort. Is that ok, or do you actually want it to be a warning + automatically convert it to `--testPathPatterns`?
```
testPathPattern:

  Option "testPathPattern" was replaced by "testPathPatterns".

  Please update your configuration.

  CLI Options Documentation:
  https://jestjs.io/docs/cli
```

a52fc2f3ce

Reply:
yeah, perfect 👍